crates/evm/src/executor/inspector/cheatcodes/fork.rs

```diff
@@ -199,7 +220,7 @@ fn create_select_fork<DB: DatabaseExt>(
 
     let fork = create_fork_request(state, url_or_alias, block, data)?;
     let id = data.db.create_select_fork(fork, data.env, &mut data.journaled_state)?;
-    Ok(id.to_ethers().encode().into())
+    Ok(DynSolValue::Uint(id, 256).abi_encode().into())
```

Comment:
Can't we go directly from `id` to the abi encode type?

Reply:
https://github.com/foundry-rs/foundry/pull/5928#discussion_r1346568383